plugins/azure-devops/README.md

```diff
@@ -313,3 +313,109 @@ To get the README component working you'll need to do the following two steps:
+
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/backend @backstage/plugin-azure-devops-common
+```
+
+New Backend you can skip the below and proceed with [permission configuration](#configure-permission)
+
+To enable permissions for the legacy backend system in `packages/backend/src/plugins/azure-devops.ts` add the following.
+
+```diff
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  return createRouter({
+    logger: env.logger,
+    config: env.config,
+    reader: env.reader,
++   permissions: env.permissions,
+  });
+}
+```
+
+### Configure Permission
+
+To apply the permission rules add the following in `packages/backend/src/plugins/permissions.ts`.
+
+> Note: the following is just an example of how you might want to setup permissions, as an Adopter you can configure this to fit your needs. Also all the permissions are Resource Permissions as they work with an Entity with the exception of `azureDevOpsPullRequestDashboardReadPermission`.
+
+```diff
+
++ import {
++  azureDevOpsPullRequestReadPermission,
++  azureDevOpsPipelineReadPermission,
++  azureDevOpsGitTagReadPermission,
++  azureDevOpsReadmeReadPermission,
++  azureDevOpsPullRequestDashboardReadPermission } from '@backstage/plugin-azure-devops-common';
++ import {
++  AuthorizeResult,
++  PolicyDecision,
++  isPermission,
++ } from '@backstage/plugin-permission-common';
++ import {
++   catalogConditions,
++   createCatalogConditionalDecision,
++ } from '@backstage/plugin-catalog-backend/alpha';
+...
+async handle(
+  request: PolicyQuery,
+  user?: BackstageIdentityResponse,
+): Promise<PolicyDecision> {
++ if ( isPermission(request.permission, azureDevOpsPullRequestReadPermission) ||
++      isPermission(request.permission, azureDevOpsPipelineReadPermission) ||
++      isPermission(request.permission, azureDevOpsGitTagReadPermission) ||
++      isPermission(request.permission, azureDevOpsReadmeReadPermission)) {
++    return createCatalogConditionalDecision(
++      request.permission,
++      catalogConditions.isEntityOwner({
++          claims: user?.identity.ownershipEntityRefs ?? [],
++       }),
++    );
++  }
+
++ if ( isPermission(request.permission, azureDevOpsPullRequestDashboardReadPermission) {
++ return {
++   result: AuthorizeResult.ALLOW,
++  };
++ }
+
+  return {
+    result: AuthorizeResult.ALLOW,
+  };
+}
+```
+
+#### Configure Azure DevOps PullRequest Dashboard Permission
+
+To Integrate Azure DevOps PullRequest Dashboard permssion, add the following in `packages/app/src/App.tsx`
+
+```diff
+...
++ import { RequirePermission } from '@backstage/plugin-permission-react';
++ import { azureDevOpsPullRequestDashboardReadPermission } from '@backstage/plugin-azure-devops-common';
+...
+...
+  const routes =(
+    <FlatRoutes>
+    ...
++   <Route
++     path="/pullrequest-dashboard"
++     element={
++      <RequirePermission permission={azureDevOpsPullRequestDashboardReadPermission}>
```

Comment:
I think this should be in the code, this saves the Adopter from having to add this and on our end we won't have to maintain this documentation.

Reply:
Yes I agree. I will update the `PullRequestPage` with `RequirePermission`